code/modules/antagonists/cult/cult_items.dm

```diff
@@ -994,8 +994,10 @@
 			if(!L.anti_magic_check())
 				if(is_servant_of_ratvar(L))
 					L.Paralyze(60)
+					break_mirror(T) //yogs added a break mirror proc
 				else
 					L.Paralyze(30)
+					break_mirror(T) //yogs added a break mirror proc
```

Comment:
Why?

Reply:
I was told it was a good nerf to do to prevent chain stun boomerang mirror spam

Reply:
a better change would be to check if they’re paralyzed and make it not able to chain stun